docs/source/tutorials/GEDI/GEDI_L4A_Example.ipynb

```diff
@@ -0,0 +1,584 @@
+{
```

Comment:
<p>After the bullet for <code>aoi</code>, we should add a bullet for <code>doi</code> as a new input:</p><p> </p><pre class="ql-syntax" spellcheck="false">- <code>doi</code>: [Digital Object Identifier] (DOI) of the GEDI collection to subset, or
&nbsp; a logical name representing such a DOI.&nbsp; Valid logical names: <code>L1B</code>, <code>L2A</code>,
&nbsp; <code>L2B</code>, <code>L4A</code>
</pre><p> </p><p>Further, remove the <strong>IMPORTANT</strong> message, because it no longer applies when using version 0.2.7 or later of the algorithm.</p><p> </p><p>I also recommend removing the 16gb recommendation. Update the recommendation to indicate only the 32gb queue.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Sure, Thanks!</p>

Reply:
<p>Sorry, I should have not included the square brackets around Digital Object Identifier. You should remove them.</p><p> </p><p>Also, <code>columns</code> and <code>query</code> no longer have defaults, so please remove the default information for those inputs.  </p>